pages/services/nifi/0.4.0-1.8.0/supported-versions/index.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: layout.pug
+navigationTitle:  Supported Versions
+title: Supported Versions
+menuWeight: 120
+excerpt: Understanding DC/OS NiFi Services package versioning
+featureMaturity:
+enterprise: false
+model: ../../data.yml
+render: mustache
+---
+
+# Supported Versions
+
+## Package Versioning Scheme
+
+    Apache NiFi: 1.5.0
```

Comment:
Why is this formatted as code? Perhaps you could format it as a bulleted list rather than indent it to make it look like code.

Reply:
done